compile.py

```diff
@@ -32,35 +35,44 @@
     "Medicine": 2700,
     "Mathematics": 2600,
     "Neuroscience": 2800,
-    "Nursing": 2900, 
+    "Nursing": 2900,
     "Pharmacology, Toxicology and Pharmaceutics": 3000,
     "Physics and Astronomy": 3100,
     "Psychology": 3200,
     "Social Sciences": 3300,
     "Veterinary": 3400}
 
+def get_file(year):
+    """Fetch and process journal impact factors for a given year."""
```

Comment:
I had this wrong, and now is a good time to fix this. Scimago Journal Ranks are not journal impact factors, which are created by Clarivate. So let's just write "Fetch an process Scimago Journal Ranks for a given year."